src/app/app.component.ts

```diff
@@ -23,6 +25,13 @@ export class AppComponent {
     this.rxStompService.watch('/chat').subscribe((message: Message) => {
```

Comment:
Should this subscription be closed? in `ngOnDestroy` maybe?

Reply:
Is indeed an idea, maybe close when no-one is subscribed and re-open is when someone is subscribing again.

However, was to lazy to implement it atm.